src/users/users.service.ts

```diff
@@ -598,24 +598,16 @@ export class UsersService {
         let userAdmin = await this.adminUserRepository.getAdminById(adminId)
         if (userAdmin) {
```

Comment:
era mais facil inverter a logica e caso o userAdmin não seja verdadeiro ja fazer um retorno reduzindo a necessidade de colocar todo o codigo dentro do if

Reply:
agora me explica esse 404 no user is admin ?

Reply:
Foi uma dica do claudio, ele pediu pra eu deixar como padrão 404, você sugere mudar?

Reply:
bem vou aprovar dps irei questionar ele sobre isso